### PR TITLE
Disable changing password when local_auth is disabled

### LINKED ISSUE #745: FeatureRequest - Disable Password Change when only using oidc
Hey,

would it be possible to disable the Button "Change Email or Password" if default['firezone']['authentication']['local']['enabled'] is set to false?

Mike

### LINKED ISSUE #750: `site.webmanifest` doesn't exist
This is an unneeded `<head>` tag and can be removed.

### PR DESCRIPTION
**Pull request by jamilbk**
Fixes #745 
Fixes #750 
Fixes firezone/product#404

### CODE REVIEW

**Comment by princemaple** [comment]
We currently have this workflow for forgot password: magic link login then change password.

Require current password will break this.

**Comment by jamilbk** [comment]
> We currently have this workflow for forgot password: magic link login then change password.
> 
> Require current password will break this.

Thanks @princemaple -- good catch. Let me work on this a bit more.

**Comment by jamilbk** [comment]
@princemaple Removed the `current_password` requirement and conditionally send a `401` if a user tries to sign in with `:identity` or `:magic_link` providers when `local_auth` is disabled.

File: apps/fz_http/lib/fz_http_web/controllers/auth_controller.ex [hunk 1]
```diff
@@ -127,16 +119,34 @@ defmodule FzHttpWeb.AuthController do
...
-        |> configure_session(renew: true)
-        |> put_session(:live_socket_id, "users_socket:#{user.id}")
-        |> Authentication.sign_in(user, %{provider: :magic_link})
-        |> redirect(to: root_path_for_role(conn, user.role))
+        maybe_sign_in(conn, user, %{provider: :magic_link})
 
       {:error, _} ->
         conn
-        |> put_flash(:warning, "The magic link is not valid or has expired.")
+        |> put_flash(:error, "The magic link is not valid or has expired.")
```
Comment (jamilbk) [inline apps/fz_http/lib/fz_http_web/controllers/auth_controller.ex:126]: Fixes an issue where this wasn't showing

**Review by princemaple — APPROVED** [review]

**Pull request merged**